webextensions/manifest/permissions.json

```
@@ -834,7 +834,7 @@
                 "version_added": false
               },
               "safari": {
-                "notes": "Does not grant an unlimited storage quota. Grants a 10 MB storage quota, instead of the standard 5 MB.",
+                "notes": "Before Safari 16, this permission granted a 10 MB storage quota.",
```

Could we specify that the storage quota is now unlimited?

```suggestion
                "notes": "Before Safari 16, this permission granted a 10 MB storage quota. Since Safari 16, the storage quota is unlimited.",
```

---

I'm going to apply this change then merge the PR.  Thank you again!